views/js/pciManager/pciManager.js

```diff
@@ -115,68 +116,106 @@ define([
                         }));
 
-                    $fileContainer.find('.switch-box').each(function(){
+                    $fileContainer.find('.switch-box').each(function () {
                         var $switch = $(this);
                         var $li = $switch.closest('.pci-list-element');
                         var typeIdentifier = $li.data('type-identifier');
                         switchFactory($switch, {
-                            on : {
-                                active : !$li.hasClass('pci-disabled')
+                            on: {
+                                active: !$li.hasClass('pci-disabled')
                             },
-                            off : {
-                                active : $li.hasClass('pci-disabled')
+                            off: {
+                                active: $li.hasClass('pci-disabled')
                             }
-                        }).on('on', function(){
+                        }).on('on', function () {
                             $li.removeClass('pci-disabled');
-                            $.getJSON(urls.enableUrl, {typeIdentifier : typeIdentifier}, function(data){
-                                if(data.success){
+                            $.getJSON(urls.enableUrl, {typeIdentifier: typeIdentifier}, function (data) {
+                                if (data.success) {
                                     listing[typeIdentifier].enabled = true;
                                     self.trigger('pciEnabled', typeIdentifier);
                                 }
                             });
-                        }).on('off', function(){
+                        }).on('off', function () {
                             $li.addClass('pci-disabled');
-                            $.getJSON(urls.disableUrl, {typeIdentifier : typeIdentifier}, function(data){
-                                if(data.success){
+                            $.getJSON(urls.disableUrl, {typeIdentifier: typeIdentifier}, function (data) {
+                                if (data.success) {
                                     listing[typeIdentifier].enabled = false;
                                     self.trigger('pciDisabled', typeIdentifier);
                                 }
                             });
                         });
                     });
 
-                    hider.show($fileContainer);
-
-                }else{
+                    $fileContainer.find('.actions').each(function () {
+                        var $unregister = $(this).find('.unregister');
+                        var $listElem = $(this).closest('.pci-list-element');
+                        var typeIdentifier = $listElem.data('type-identifier');
+                        buttonFactory({
+                            id: 'unregister',
+                            type: 'info',
+                            icon: 'bin',
+                            label: 'Delete',
```

Comment:
Don't forget to translate UI labels so they work in all languages: `label: __('Delete')`.

(This is why the 'i18n' helper is included in most js files)